src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -234,6 +234,15 @@ private static IEnumerable<ISetupInstance> EnumerateVisualStudioInstances()
         private static ISetupInstance LocateVisualStudioInstance(ImmutableHashSet<string> requiredPackageIds)
         {
             var vsInstallDir = Environment.GetEnvironmentVariable("VSInstallDir");
```

Comment:
:memo: This is not set when run from Test Explorer. I saw a value `VSAPPIDDIR` set, but I wasn't sure what that was. The value used below seemed to be the obvious one to handle this case.

Reply:
If that translates to "visual studio app ID directory", that might indeed be the right thing to use. Not sure who to confirm that with though.